src/Compute/Compute.Test/ScenarioTests/VirtualMachineExtensionTests.cs

```diff
@@ -87,36 +87,24 @@ public void TestAzureDiskEncryptionExtension()
             TestRunner.RunTestScript("Test-AzureDiskEncryptionExtension");
         }
 
-#if NETSTANDARD
```

Comment:
These tests used to run in NetCore, and run only in Desktop runs on Netstandard - now they will not run in NetCore or NetStandard.  Please revert so these tests can continue to run in NoetCore

Reply:
@markcowl  The current change has [Fact] for both NetCore and Netstandard